src/main/java/com/google/devtools/build/lib/vfs/FileSystemUtils.java

```diff
@@ -917,19 +881,27 @@ private LongReadIOException(Path path, int fileSize) {
    * the number of bytes read.
    *
    * <p>Use this method when you already know the size of the file. The check is intended to catch
-   * issues where filesystems or external interference results in truncated or concurrently modified
-   * files.
+   * issues where filesystems or external interference results in truncated or concurrent
```

Comment:
Nit [clarity]: Imo something like the following more clearly conveys the intended meaning while keeping your word choice. This is an optional suggestion, so feel free to ignore!

  where the filesystem incorrectly returns truncated file contents, or where an external modification has concurrently truncated or appended to the file.

Reply:
Sure

@meteorcloudy fyi since I'm not sure who is going to be mailed the internal CL

Reply:
gTech will do the import, but I can suggest the edition, np!

Reply:
Or I can just edit this PR, done ;)